src/HX711Multi.py

```diff
@@ -0,0 +1,238 @@
+from typing import List
+import RPi.GPIO as GPIO
+import numpy as np
+
+"""
+
+HX711_Multi: This class provides a simple way to interface with multiple
+HX711 digital to analog converters connected on a single clock line.
+It's intended use is for the strain gauge PCB in Xenia-1
+
+Based on Shmulike's Arduino version https://github.com/shmulike/HX711-multi
+
+
+Attributes
+----------
+
+__PD_SCK_pin [int]: GPIO pin connected to all the clocks of each HX711
+(Pin 11 labeled PD_SCK in datasheet)
+
+__DOUT_pins List[int]: GPIO pins connected to data pins of each HX711
+(Pin 11 labeled DOUT in datasheet)
+
+__gain [int]: HX711 features programmable gain settings of 64 or 128 for
+channel A which is what we use. 128 is used by default. NOTE: __gain stores
+the number of cycles of the PD_SCK pin needed to set the gain for the HX711. For
+128, __gain = 3, for 64, __gain = 2. This is outlined in the datasheet for the
+HX711.
+
+__debug_enabled [bool]: Print debug messages to the console when True.
+
+__offsets List[int]: Store the offsets generated by the tare(). This is
+the same as tare a scale so that it reads 0 with nothing on it.
+
+__timeout_ms [int]: Max time in milliseconds to wait for HX711s to become
```

Comment:
Nice catch. Also I realized I don't actually use __timeout anywhere. I wasn't sure the best way to implement it. Basically right now if an HX711 disconnects or something, it will never become ready and this will become an infinite loop. I don't want to use a time.sleep() because that's poor practice.

I guess I could count the number of attempts to become ready, and if it doesn't after say 200 fail? What do you think?